deploy/helm/sumologic/README.md

```diff
@@ -17,6 +17,7 @@ Parameter | Description | Default
 `sumologic.setupEnabled` | If enabled, a pre-install hook will create Collector and Sources in Sumo Logic. | `true`
 `sumologic.logs.enabled` | Set the enabled flag to false for disabling logs ingestion altogether. | `true`
 `sumologic.metrics.enabled` | Set the enabled flag to false for disabling metrics ingestion altogether. | `true`
+`sumologic.traces.enabled` | Set the enabled flag to true to enable tracing ingestion. | `false`
```

Comment:
Perhaps we should add a note like "Tracing must be enabled for the account first. Please contact your Sumo representative for activation details" - is it a fit place @frankreno ? We don't have really good handling of the error when source creation fails due to tracing not being enabled (going to wrap Terraform exception but that's all that can be done easily without more significant changes). So maybe at least such note could help?

Reply:
Yeah, that's a good idea! Please add it here.